test/Containerfile.e2e

```diff
@@ -0,0 +1,16 @@
+FROM --platform=linux/amd64 mcr.microsoft.com/oss/go/microsoft/golang:1.24-fips-cbl-mariner2.0@sha256:374281fe968134a44bcc7bd4b9e2d55dda61dcaa942fee5dda28cd38d54619cd as builder
+
+WORKDIR /app
+COPY --from=ARO-HCP test ./test
+COPY --from=ARO-HCP internal ./internal
+RUN touch junit-report.xml
+RUN cd test/e2e && make e2etest
+
+FROM --platform=linux/amd64 mcr.microsoft.com/cbl-mariner/distroless/base:2.0-nonroot@sha256:a5d5928601847a40c81fa397df33136c285866c974bf91ed94d688390fcc33f3
+
+WORKDIR /
+ENV SETUP_FILEPATH=e2e-setup.json
+COPY --from=builder /app/test/e2e/$SETUP_FILEPATH $SETUP_FILEPATH
```

Comment:
I would not include the setup file into a container, but would rather pass it via a volume (see docker/podman --volume).

This means that the container would assume that a file is present on given path, but it will be passed only when the test is executed later, not during a build.

Reply:
It sounds like we need a check somewhere (probably out of scope of this PR) to make sure we have the appropriate e2e-setup file already exists.